File: beecrowd/1011 - Esfera.go
package main

import (
	"fmt"
	"math"
)

type Esfera struct {
	raio float64
}

func NovaEsfera(raio float64) *Esfera {
	return &Esfera{
		raio: raio,
	}
}

func (e *Esfera) volume() float64 {
	const PI = 3.14159
	return 4.0 / 3.0 * math.Pow(e.raio, 3) * PI
}

func main() {

	var raio float64

	fmt.Scan(&raio)

	esfera := NovaEsfera(raio)

	fmt.Printf("VOLUME = %.3f\n", esfera.volume())
}

File: beecrowd/1011 - Esfera.kt
import java.util.*
import kotlin.math.pow

fun volumeEsphere(r: Double): Double {
    val PI = 3.14159

    return r.pow(3) * (4.0 / 3.0) * PI
}

fun main(args: Array<String>) {

    val radius = readLine()!!.toDouble()

    println("VOLUME = %.3f".format(Locale.US, volumeEsphere(radius)))
}